### PR TITLE
Add OpenJDK 15 support

### PR DESCRIPTION
**Pull request by SueChaplain**
- supported environments
- AdoptOpenJDK topic
- Introduction - build

Closes:  #603

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
@pshipton - please advise if there are any platform support changes or compiler changes for OpenJDK 15.

**Comment by pshipton** [comment]
There are OpenJ9 changes in the works, but nothing completed yet. For now, please remove support for CentOS and RHEL 6 for Java 15. Adopt has finished with this change. https://github.com/AdoptOpenJDK/openjdk-build/pull/1865

https://github.com/eclipse/openj9/issues/9805
https://github.com/eclipse/openj9/issues/9855

File: docs/openj9_support.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ OpenJDK 11 binaries are supported on the minimum operating system levels shown i
 When public support for an operating system version ends, OpenJ9 can no longer be supported on that level.
 
 
-### OpenJDK 14
+### OpenJDK 15
```
Comment (pshipton) [inline docs/openj9_support.md:150]: There is a "OpenJDK 14" a few lines down.

Comment (SueChaplain) [reply docs/openj9_support.md:150]: Fixed

**Comment by SueChaplain** [comment]
@pshipton - Updated the PR to remove Centos/RHEL 6 for  OpenJDK 15, and fixed the missed reference to 14. Do you want us to keep this PR open until the other Adopt changes land or just make updates later on. Feel free to merge.

**Comment by pshipton** [comment]
We can make further updates later when appropriate.

**Pull request merged**